container-search/src/test/java/com/yahoo/prelude/query/test/SameElementItemTestCase.java

```diff
@@ -0,0 +1,35 @@
+package com.yahoo.prelude.query.test;
+
+import com.yahoo.prelude.query.AndItem;
+import com.yahoo.prelude.query.SameElementItem;
+import com.yahoo.prelude.query.WordItem;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class SameElementItemTestCase {
+    @Test
+    public void testAddItem() {
+        SameElementItem s = new SameElementItem("structa");
+        s.addItem(new WordItem("b", "f1"));
+        s.addItem(new WordItem("c", "f2"));
+        s.addItem(new WordItem("d", "f3"));
+        assertEquals("SAME_ELEMENT f1:b f2:c f3:d", s.toString());
```

Comment:
consider letting toString also mention the struct name

Reply:
Fixed.